teku/src/main/java/tech/pegasys/teku/cli/options/MetricsOptions.java

```diff
@@ -63,10 +63,9 @@
   private Set<MetricCategory> metricsCategories = DEFAULT_METRICS_CATEGORIES;
 
   @Option(
-      names = {"--metrics-host-whitelist"},
+      names = {"--metrics-host-allowlist", "--metrics-host-whitelist"},
       paramLabel = "<hostname>",
-      description =
-          "Comma separated list of hostnames to whitelist for access, or * to accept any host",
+      description = "Comma separated list of hostnames to allow, or * to allow any host",
       split = ",",
       arity = "0..*")
   private final List<String> metricsHostWhitelist = Arrays.asList("127.0.0.1", "localhost");
```

Comment:
rename variable now, and any getter, config item in TekuConfiguration etc.

Reply:
done